index.js

```diff
@@ -69,10 +67,6 @@ function createHandler(filename, reports, phantom, mockserverHandler) {
         files = files.map(normalizePath);
         files.unshift(path.join(__dirname, '/lib/override-log.js'));
```

Comment:
I think there might be a better way to get logs from puppeteer than this, but it's ok to leave it as is for now.